### ISSUE TITLE
NVDA should announce opening/closing a tab in IE or FF

### ISSUE THREAD

**Issue by nvaccessAuto** [nvaccess/nvda#1490]
**Reported by allan on 2011-05-05 08:38**
Since NVDA is a screen reader, in my opinion it should tell the user whether it has a new tab is open or closing in IE of FF.

E.g.
1. User has press CTRL+enter in a link, it will open a new tab, but NVDA stay silent.
1. User press CTRL+F4 to close a tab, but NVDA doesn't notify the user.

**Comment by bhavyashah** [comment]
CC @michaelDCurran and @josephsl (for Edge) and @MarcoZehe (for Firefox) to seek browser-specific or more universal ways of implementing the requested feature.

**Comment by Adriani90** [comment]
I think announcing new tab or tab closed every time when using the commands will introduce too much verbosity in NVDA. In general, NVDA reads the title of the new page when you press ctrl+enter on a link and announces the title of the page when you press ctrl+w to close the tab.

**Comment by Adriani90** [comment]
The only issue I see here is ctrl+enter for opening a link in a new tab.In this case most of times NVDA does not say anything.

**Comment by Adriani90** [comment]
I am closing this in favor of #9625
